dask/dot.py

```diff
@@ -1,9 +1,8 @@
 from __future__ import absolute_import, division, print_function
 
-from subprocess import check_call
-
 from graphviz import Digraph
-
+from subprocess import check_call
+import re
```

Comment:
Just a small opinionated pep8 gripe here:


>Imports should be grouped in the following order:
>
>    - standard library imports
>    - related third party imports
>    - local application/library specific imports
>
>You should put a blank line between each group of imports. 

This is why they were organized that way originally. Usually I'm not a stickler for pep8 things, but import organization really bothers me :)